SD_Assignment1/target/classes/db/migration/V1__InitialSchema.sql

```diff
@@ -14,6 +26,22 @@ CREATE TABLE IF NOT EXISTS users
 	
 );
 
+CREATE TABLE IF NOT EXISTS qvotes
+(
+	voteid serial PRIMARY KEY,
+	questionid int NOT NULL,

```

Comment:
You don't have any foreign keys in your tables. You should add keys between tables to ensure referential constraints.